labman/db/tests/tester.py

```diff
@@ -216,20 +216,13 @@ def integration_tests():
     samples = get_samples()
```

Comment:
Just want to be sure that you want to delete vs. comment out these lines ... we are gonna need them back some time soon ...

Reply:
Fair point!  I've put the lines to run the comparison back in but not the actual regex against which the comparison is done, as that will have to change pretty much from the ground up when we get around to bringing this feature back.